Makefile

```diff
@@ -69,6 +69,9 @@ endif
 
 DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
+# shell command used for `make shell`
+SH := $(if $(SH),$(SH),bash)
```

Comment:
Don't they do extra processing on the value in either the true or false case? (adding prefix/suffix or the like)

I suppose keeping consistent is fair enough though. :+1: